ui/raidboss/data/06-ew/trial/endsinger.txt

```diff
@@ -0,0 +1,100 @@
+375.9 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+388.0 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+401.2 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+407.2 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+417.3 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+426.4 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+436.6 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+446.2 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+450.8 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+464.0 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+483.2 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+490.3 "--untargetable--"
+499.6 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
+501.8 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
+507.8 "--targetable--"
+513.9 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+519.9 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+539.2 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+549.4 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+559.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+563.5 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+576.7 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+
+# Phase 2, add spawns
+1000.0 "Planetes" sync / 1[56]:[^:]*:The Endsinger:6B58:/ window 1000,0
+1000.0 "--untargetable--"
+1014.3 "--targetable--"
+1023.4 "Meteor Radiant" sync / 1[56]:[^:]*:Kakodaimon:6656:/
+1035.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1040.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1045.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1050.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1056.6 "Meteor Outburst" sync / 1[56]:[^:]*:Kakodaimon:665C:/
+# Loop Katastrophe raidwide until enrage
+1067.8 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
+1078.9 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/ jump 1067.8
+1090.0 "Katastrophe"
+1101.1 "Katastrophe"
+1112.2 "Katastrophe"
+1123.3 "Katastrophe"
+1200.0 "--sync--" sync / 00:0039::10 seconds until oblivion is sealed/ window 200,0
+1212.2 "Ultimate Fate" sync / 1[56]:[^:]*:The Endsinger:6B59:/ window 10,10
+1218.9 "--targetable--"
+
+# Phase 3, loops until soft enrage
+2000.0 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6B5B:/ window 1000,0
+2036.0 "--untargetable--"
+2074.0 "--targetable--"
+2086.1 "Telos" sync / 1[56]:[^:]*:The Endsinger:665E:/
+2099.3 "Telomania (cast)" sync / 1[56]:[^:]*:The Endsinger:665F:/ window 0,30
+2103.6 "Telomania (small) x4" duration 5 #sync / 1[56]:[^:]*:The Endsinger:6660:/
+2114.6 "Telomania (big)" sync / 1[56]:[^:]*:The Endsinger:6661:/
+2120.1 "Dead Star x3" duration 6 #sync / 1[56]:[^:]*:The Endsinger:5E4E:/
+2124.5 "Telomania (cast)" sync / 1[56]:[^:]*:The Endsinger:665F:/ window 0,30 jump 2099.3
+2128.8 "Telomania (small) x4" duration 5 #sync / 1[56]:[^:]*:The Endsinger:6660:/
+2139.8 "Telomania (big)" #sync / 1[56]:[^:]*:The Endsinger:6661:/
+2145.3 "Dead Star x3" duration 6 #sync / 1[56]:[^:]*:The Endsinger:5E4E:/
```

Comment:
Can you add a loop block underneath dead star, rather than a huge sync on Fatalism? Otherwise, the timeline will be blank and then suddenly fill in again.  It's also not clear how much time there is between Dead Star and the next use of Fatalism.  (Maiko's extension in VSCode has a "set time to", where you can copy this whole block, select the copy, and then right click and say "set time to", and it will set the first time and adjust everything after it to easily make a loop block.)

e.g. https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.txt#L74-L79

Reply:
I think this *is* the loop block, it's just not delineated with whitespace. There should probably be a blank line between 2086.1/2099.3, and between 2120.1/2124.5. The Fatalism sync is on the other side of the block.

Reply:
Oh, you're right, thanks!